frontend/src/tests/components/Reports/ReportHeaderTable.test.js

```diff
@@ -22,7 +23,7 @@ describe("ReportHeaderTable tests", () => {
     const expectedFields = [ 'cowPrice', 'milkPrice', 'startingBalance', 'startingDate', 'showLeaderboard', 'carryingCapacity', 'degradationRate', 'belowCapacityHealthUpdateStrategy', 'aboveCapacityHealthUpdateStrategy']
     const expectedHeaders = ['Cow Price', 'Milk Price', 'Start Bal', 'Start Date', 'Leaderboard', 'Capacity', 'Degrad Rate', 'BelowCap', 'AboveCap']
 
-    const testId = "ReportHeaderTable"
+    // const testId = "ReportHeaderTable"
```

Comment:
why not just delete this since it was moved

Reply:
Agreed with Katy: see: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html